src/stories/components/Branding/Branding.js

```diff
@@ -0,0 +1,50 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+// TODO: replace the import below with the actual logo once it's chosen
+import Logo from '../../../images/Polygon.png';
+
+/**
+ * Component for branding element.
+ *
+ * @component
+ * @param {string} modifierClasses Class modifiers of the component.
+ * @param {string} text Text of the brand.
+ * @return {object} (
+ *   <Branding modifierClasses={modifierClasses}
+ *             text={text} />
+ * )
+ */
+export const Branding = ({ modifierClasses, logo, text }) => {
+    return (
+        <a
+            href="*/"
+            className={['branding', `${modifierClasses}`].join(' ').trim()}
+        >
+            <img src={Logo}
+                alt={`${text} website logo`}
+                className="logo"/>
+            {text}
```

Comment:
I decided to just use the text inside the anchor tag (<a><a/>) as the brand name. I wonder if it would be better to use a header tag instead (such as <h1><h1/>)...

Reply:
Not sure if one is better than the other, but if either one is actually better, I'm sure it should be a very easy fix.

Reply:
You cannot nest block elements (`<h1>`, `<p>`, etc) inside inline elements (like `<a>`, `<span>`, etc). This is okay for now!